src/pybind/mgr/dashboard/controllers/nvmeof.py

```diff
@@ -37,24 +36,19 @@ def create(self, rbd_pool: str, rbd_image: str, subsystem_nqn: str,
             :param create_image: Create RBD image
             :param image_size: RBD image size
             :param block_size: NVMeoF namespace block size
-            :param nsid: NVMeoF namespace ID
-            :param uuid: NVMeoF namespace UUID
-            :param anagrpid: NVMeoF namespace ANA group ID
             """
             response = NVMeoFClient().create_namespace(rbd_pool, rbd_image,
                                                        subsystem_nqn, block_size,
-                                                       nsid, uuid, anagrpid,
                                                        create_image, image_size)
             return json.loads(MessageToJson(response))
 
         @Endpoint('DELETE', path='{subsystem_nqn}')
-        def delete(self, subsystem_nqn: str, nsid: int):
+        def delete(self, subsystem_nqn: str):
```

Comment:
BTW this doesn't have the `@DeletePermission`. Is that intended?

Reply:
nope, definitely not. Let me fix those. thanks for catching

Reply:
updated!